apps/docs/stories/architecture-decision-records/0013-why-using-virtual-carousel-item.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs'
+
+<Meta title="Docs/Architecture Decision Records/ADR 0013: Why using a virtual carousel item" />
+
+# ADR 0013: Why using a virtual carousel item
+
+🗓️ 2023-08 · ✍️ [@schirrel](https://github.com/schirrel), [@MayaraRMA](https://github.com/MayaraRMA), [@tassioFront](https://github.com/tassioFront) and [@felipefialho](https://twitter.com/felipefialho_)
+
+## Context
+Wrapping up Swiper HTMLElements as Atomium element.
+
+## Problems
+When creating `<atom-carousel>` and its items `<atom-carousel-item>`, the HTML three place the `<atom-carousel-item>` inside `<swiper-element>`, but the swiper needs its children to be `<swiper-slide>`.
+
+Then we need to convert each `atom-carousel-item` into `swiper-slide`.
+
+## Considered Options
+To convert the inner `<atom-carousel-item>` to `<swiper-slide>`, and we have two options:
+
+- Create and `atom-carousel-item` component and inside itself creating the `<swiper-slide>` component on parent and them self deleting from parent.
+- The parent itself convert all children at once.
+
+## Decision Drivers
+
+- Complexity
+- Performance
+
+### Performance
+We had performed a test with both approach using 9 items inside the Carousel:
+- With children `atom-carousel-item` doing the conversion, it took _0.50ms_
+- With parent `atom-carousel` doing the conversion, it took  _0.20ms_
+
+
+### Complexity
```

Comment:
Question: We are considering time complexity only, does make sense to specify it?

Reply:
Basically only @tassioFront, look for O complexity in both cases is the same, one approach the loop is on the parent, other approach has no explicity loop, but he operation repeats the same amount as children (inside of each).
The other operations are simple 1.

Besides on the parent we just have a simple string Mmanagement  and remove child.  On the children we must create a structured component, append to parent and delete the old.

Do you see that this level of details is needed?

Reply:
Sorry, I misunderstood the text order. I saw the time comparison time, but it is in the `Performance` block, not the `Complexity` one. I don't need it is required. Keep going, my friend